pkg/dev/podmandev/reconcile.go

```diff
@@ -118,6 +123,24 @@ func (o *DevClient) reconcile(
 	return nil
 }
 
+// warnAboutK8sResources prints a warning if the Devfile contains a K8s resource that it needs to create on Podman.
+func (o *DevClient) warnAboutK8sResources(devfileObj parser.DevfileObj) {
+	k8sComponents, err := devfile.GetKubernetesComponentsToPush(devfileObj, false)
+	if err != nil {
+		return
+	}
+	for _, comp := range k8sComponents {
+		uList, err := libdevfile.GetK8sComponentAsUnstructuredList(devfileObj, comp.Name, "", devfilefs.DefaultFs{})
+		if err != nil {
+			continue
+		}
+		for _, u := range uList {
+			log.Warningf("Skipping Kubernetes component \"%v/%v\". Kubernetes components are not supported on Podman.", u.GetKind(), u.GetName())
```

Comment:
This message might be repeated if there are multiple resources.
I'd suggest we display a single warning with all the resource names as a comma-separated string (or a bullet list). What do you think?

Reply:
Also, I'm wondering if we need to go so far with the K8s component (via `libdevfile.GetK8sComponentAsUnstructuredList`). I think it is not needed as we skipping it anyways. Worse, the warning will not be displayed if `libdevfile.GetK8sComponentAsUnstructuredList` returns an error (if the YAML content for the Kubernetes component is invalid for example).
So I think we could just display the component name as defined in the Devfile (`comp.Name`)). What do you think?